code/parse/sexp.cpp

```diff
@@ -33765,6 +33797,14 @@ SCP_vector<sexp_help_struct> Sexp_help = {
 		"\t2:\t(Optional) Ignore the directive count set by any earlier SEXPs in the event. If set to false it will add instead\r\n"
 	},
 
+	// wookieejedi
+	{ OP_GET_HOTKEY, "get-hotkey\r\n"
+		"\tReturns the hotkey integer of the ship or wing.\r\n"
+		"\tNo hotkey returns -1, F5 returns 0, F6 returns 1, ...\r\n\r\n"
```

Comment:
You should indicate what the maximum possible returned value is, and what it corresponds to.